rope/refactor/patchedast.py

```diff
@@ -394,7 +394,7 @@ def _Constant(self, node):
             self._handle(node, ["..."])
             return
 
-        assert False
+        raise AssertionError(f"unknown node.value: {node.value!r}")
```

Comment:
This isn't a valid transformation. `assert ...` has a very different semantic than `raise AssertionError`.

`assert` is a comment saying that a certain invariant holds true; and `assert ...` is saying that line is meant to be unreachable. `raise ...` implies that there is a possibility of this being a valid code path, which just happens to raise an exception, which isn't the case here. They have very different intent.

Just because it behaves the same in runtime doesn't mean that it has the same semantic. linters are more often than not just very wrong on this kind of things, so I'd suggest we just turn off the linter warnings on assert.

An alternative could be to use `typing.assert_never()`, but the way the `ast` handle Constant nodes, type checkers isn't very useful here.

Reply:
@lieryan I like your reasoning!  I'll update this PR accordingly:

- Restore the original asserts.
- Suppress `B011: Do not call assert False. Instead raise AssertionError()` in setup.cfg.